src/Shared/Recruit.Vacancies.Client/Domain/Entities/ProgrammeLevelHelper.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace Esfa.Recruit.Vacancies.Client.Domain.Entities
+{
+    public static class ProgrammeLevelHelper
+    {
+        public static bool TryRemapFromInt(int value, out ProgrammeLevel result)
+        {
+            switch (value)
+            {
+                case 5:
```

Comment:
I assume this is more permanent as long as the rule exists to make the modification. Reading this class and understanding the particular change from x to y is difficult. Could you please put `case 5: // PhD` or whatever 5 was supposed to be and the same for `case 7:` please.